### DISCUSSION TITLE
打开数据库设置的时候报错

### DISCUSSION THREAD

**Discussion by liuzedong-hub** [1Panel-dev/1Panel#6181] [Q&A]
![image](https://github.com/user-attachments/assets/ee73d308-0bc6-4569-805e-c72b1eb37198)

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，已知问题 https://github.com/1Panel-dev/1Panel/issues/6148

**Reply by liuzedong-hub** [reply]
感谢
